packages/app/src/app/pages/Sandbox/QuickActions/QuickActions.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
```

Comment:
Even if you rename the file, you should have an index.ts file for exporting the component, so you dont have two add extra file name in the other two files of the PR

Reply:
I've done the changes, please check